src/modules/acc/acc_logic.c

```diff
@@ -372,6 +380,9 @@ void acc_onreq( struct cell* t, int type, struct tmcb_params *ps )
 	int tmcb_types;
 	int is_invite;
 
+	LM_ERR("is_mc_on: %d\n", is_mc_on(ps->req));
+	LM_ERR("is_acc_on: %d\n", is_acc_on(ps->req));
+	LM_ERR("is_acc_prepare_on: %d\n", is_acc_prepare_on(ps->req));
```

Comment:
There are couple of error messages which I guess are only for testing purposes. Before merging should be removed. I could spot in two other places, besides the three here.